presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/StatisticsFetcher.java

```diff
@@ -17,11 +17,11 @@
 import com.github.dockerjava.api.model.MemoryStatsConfig;
 import com.github.dockerjava.api.model.StatisticNetworksConfig;
 import com.github.dockerjava.api.model.Statistics;
-import com.github.dockerjava.core.InvocationBuilder;
 import io.airlift.log.Logger;
 import io.airlift.units.DataSize;
 import net.jodah.failsafe.FailsafeExecutor;
 import org.testcontainers.DockerClientFactory;
+import org.testcontainers.shaded.com.github.dockerjava.core.InvocationBuilder;
```

Comment:
Why did this change? We shouldn't have dependencies on shaded classes, as they are an implementation detail of the library that uses them.

Reply:
Agreed, but the package com.github.dockerjava.core no longer exists.  All I want is to be able to run product-tests again.  Suggestions?